lib/transactions/multisignature.js

```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating multisignature in Lisk Blockchain
```

Comment:
Multisignatures module provides functions for creating multisignature transactions for use with the Lisk Blockchain